docs/source/platform_technical_documentation/ade_custom_extensions/jupyter_server.ipynb

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
<p>Recommend:</p><ul><li>Capitalizing instances of "jupyter"</li><li>Changing "endpoints supported" to "supported endpoints"</li><li>Changing "the specific endpoints refer to the" to "the specific endpoints, refer to the"</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>